NppMenuSearch/Main.cs

```diff
@@ -45,8 +46,9 @@ internal static void CommandMenuInit()
             PluginBase.SetCommand(0, "Menu Search...", MenuSearchFunction, new ShortcutKey(true, false, false, Keys.M));
             PluginBase.SetCommand(1, "Clear “Recently Used” List", ClearRecentlyUsedList, new ShortcutKey(false, false, false, Keys.None));
             PluginBase.SetCommand(2, RepeatPreviousCommandLabel, RepeatLastCommandFunction, new ShortcutKey(false, false, false, Keys.None));
-            PluginBase.SetCommand(3, "---", null);
-            PluginBase.SetCommand(4, "About...", AboutFunction, new ShortcutKey(false, false, false, Keys.None));
+            PluginBase.SetCommand(3, "Dark Mode", DarkModeFunction, new ShortcutKey(false, false, false, Keys.None), IsDarkModeEnabled);

```

Comment:
There is only a Notepad++ message for plugins to get the background color of the editor (NPPM_GETEDITORDEFAULTBACKGROUNDCOLOR), that I know of to detect dark theme. So if we'd use this to detect the dark theme and a user would use a dark editor and light theme, it would look like this: ![grafik](https://user-images.githubusercontent.com/6975881/150698520-1cc8b8b5-bf89-4c7c-a141-d846dd9695fc.png)

Reply:
With the next version of Notepad++ Dark Mode detection for PlugIns will be added (https://github.com/notepad-plus-plus/notepad-plus-plus/pull/11547). I'll take a look, how to implement this.